### PR TITLE
Single level bom as built 1.0.0

### LINKED ISSUE #78: [New Model]: [Traceability] New model SingleLevelBomAsBuilt (replacement of AssemblyPartRelationship)
## Model Description
We want to create a new model named SingleLevelBomAsBuilt which will replace the existing model AssemblyPartRelationship. The reason for the replacement is to match our naming convention from BomAsPlanned and other BoMs. 

## Relevant Standards
New model will replace existing AssemblyPartRelationship, therefore not appicable

## Example Data
New model will replace existing AssemblyPartRelationship, therefore not appicable

## MS1 Criteria

- [ ] The proposed aspect model does not exist already
- [ ] The proposed aspect model does not extend an existing aspect but introduces completely new functionality
- [ ] The proposal references a Catena-X use case
- [ ] Relevant standards are mentioned/linked
- [ ] A first draft of the data structure is provided in an example

### PR DESCRIPTION
**Pull request by jacewski-bosch**
## Description

Renamed Aspect for more clarity and following the naming convention as per other tracebility aspects.
Additionally to changes have been introduced with this major update:

* property naming now follows data chain standards
* BPN of child item supplier has been added to allow for efficient EDC and Asset shell discovery in decentralized digital twin registry environments.

Closes #78 

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the BAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar bamm-cli.jar -i \<path-to-aspect-model\> -v ). The  BAMM CLI is available [here](https://openmanufacturingplatform.github.io/sds-documentation/sds-developer-guide/dev-snapshot/tooling-guide/bamm-cli.html) and in [GitHub](https://github.com/OpenManufacturingPlatform/sds-sdk/releases)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "name" and "description"** in English language. 
- [x] **no duplicate names or preferredNames** within an Aspect (e.g. a Property and the referenced Characteristic should not have the same name)
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the BAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/v1.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'

### CODE REVIEW

File: io.catenax.single_level_bom_as_built/1.0.0/SingleLevelBomAsBuilt.ttl [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.single_level_bom_as_built:1.0.0#> .
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
```
Comment (bs-jokri) [inline io.catenax.single_level_bom_as_built/1.0.0/SingleLevelBomAsBuilt.ttl:35]: Remove one of the blocks

File: io.catenax.single_level_bom_as_built/1.0.0/SingleLevelBomAsBuilt.ttl [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+   samm:exampleValue "BPNL50096894aNXY" .
+
+:QuantityCharacteristic a samm-c:Quantifiable ;
+   samm:description "Describes the quantity in which the child part is assembled in the given parent object by providing a quantity value and the measurement unit in which the quantity is measured."@en ;
+   samm:dataType :Quantity .
+
+:LifecycleContextCharacteristic a samm-c:Enumeration ;
+   samm:preferredName "Lifecycle Context"@en ;
+   samm:description "The possible values for lifecycle context as defined by STEP: AsRequired, AsDesigned, AsPlanned, AsBuilt, AsMaintained, AsRecycled"@en ;
+   samm:see <http://private.pdm-if.org/web/pdm-if/recommended-practices1> ;
```
Comment (bs-jokri) [inline io.catenax.single_level_bom_as_built/1.0.0/SingleLevelBomAsBuilt.ttl:125]: die URL scheint nicht zugreifbar zu sein

File: io.catenax.single_level_bom_as_built/1.0.0/SingleLevelBomAsBuilt.ttl [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+:measurementUnit a samm:Property ;
+   samm:description "Unit of Measurement for the quantity of serialized objects"@en ;
+   samm:see <https://eclipse-esmf.github.io/samm-specification/2.0.0/appendix/unitcatalog.html> ;
+   samm:characteristic samm-c:UnitReference ;
+   samm:exampleValue "unit:litre"^^samm:curie .
+
+:NumberofObjects a samm:Characteristic ;
+   samm:description "Quantifiable number of objects in reference to the measurementUnit"@en ;
+   samm:dataType xsd:double .
+
```
Comment (bs-jokri) [inline io.catenax.single_level_bom_as_built/1.0.0/SingleLevelBomAsBuilt.ttl:164]: Release notes are missing (just copy from another model and adjust accordingly). Please also indicate where the model originates from

**Review by bs-jokri — CHANGES_REQUESTED** [review]
please check my comments

File: io.catenax.single_level_bom_as_built/1.0.0/SingleLevelBomAsBuilt.ttl (hunk 1 above)
Comment (jacewski-bosch) [reply io.catenax.single_level_bom_as_built/1.0.0/SingleLevelBomAsBuilt.ttl:35]: done.

File: io.catenax.single_level_bom_as_built/1.0.0/SingleLevelBomAsBuilt.ttl (hunk 2 above)
Comment (jacewski-bosch) [reply io.catenax.single_level_bom_as_built/1.0.0/SingleLevelBomAsBuilt.ttl:125]: after discussing with MS3 approvers, I removed the entire property.

**Comment by jacewski-bosch** [comment]
modeling_team

**Comment by bs-jokri** [comment]
SAMM meta model will be change to BAMM 2.0.0 prior release

**Pull request merged**